src/main/java/org/web3j/tx/HighThroughputTransactionManager.java

```diff
@@ -0,0 +1,95 @@
+package org.web3j.tx;
+
+import org.web3j.crypto.Credentials;
+import org.web3j.crypto.Hash;
+import org.web3j.crypto.TransactionEncoder;
+import org.web3j.protocol.Web3j;
+import org.web3j.protocol.core.DefaultBlockParameterName;
+import org.web3j.protocol.core.methods.request.RawTransaction;
+import org.web3j.protocol.core.methods.response.EthGetTransactionCount;
+import org.web3j.protocol.core.methods.response.EthSendTransaction;
+import org.web3j.protocol.exceptions.TransactionTimeoutException;
+import org.web3j.utils.Numeric;
+
+import java.math.BigInteger;
+import java.util.HashMap;
+import java.util.concurrent.ExecutionException;
+
+/**
+ * Created by fernando on 2/1/17.
```

Comment:
Please can you use the Javadoc header to include a simple class description? Your name will be visible from the commit log, so it's not required here.